<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color: white;
color: rgb(0,120,0);
font: bold 12pt &quot;Roboto&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Container" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QLabel" name="backgroundPic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>600</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../propertyPics/hotel.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="innerContainer" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>750</width>
       <height>520</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#innerContainer{
background-color: rgba(255, 255, 255, 200);
border-radius: 20px;
border: 3px solid rgb(0,120,0);
}</string>
     </property>
     <widget class="QLabel" name="lblUser">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>20</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../icons/icons.qrc">:/icons/user (2).png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblLogo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>120</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../icons/icons.qrc">:/newPrefix/DEVOE partners logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbxType">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Property Type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>House</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Flat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Plot File</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Commercial Plot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Residential Plot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Agricultural Plot</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cmbxCity">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>City</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lahore</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Islamabad</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rawalpindi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Karachi</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sialkot</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gujranwala</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cmbxPurpose">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>210</y>
        <width>200</width>
        <height>35</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Purpose</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>For Sale</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>For Rent</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cmbxArea">
      <property name="geometry">
       <rect>
        <x>489</x>
        <y>280</y>
        <width>201</width>
        <height>35</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Area</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 Marla</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5 Marla</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10 Marla</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 Kanal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 kanal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 Kanal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5 Kanal</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: None;</string>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnSearch">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>340</y>
        <width>500</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 120, 0);
color: white;
font: bold 16pt &quot;Roboto&quot;;
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnExit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>450</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnShowAll">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>450</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Show All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Heading">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>140</y>
        <width>500</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 120, 0);
color: white;
font: bold 14pt &quot;Roboto&quot;;
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>Search properties for sale in Pakistan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnAdd">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>25</y>
        <width>161</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
color: rgb(0, 120, 0);
font: bold 12pt &quot;Roboto&quot;;
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>Add Property</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPriceFrom">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>280</y>
        <width>170</width>
        <height>35</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>From:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPriceTo">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>280</y>
        <width>170</width>
        <height>35</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>To:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblUserName">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>90</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,120,0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="BtnScrap">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>25</y>
        <width>160</width>
        <height>50</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
color: rgb(0, 120, 0);
font: bold 12pt &quot;Roboto&quot;;
border-radius: 20px;</string>
      </property>
      <property name="text">
       <string>Scrap Data</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
